Sources/Sharing/Internal/Reference.swift

```diff
@@ -339,10 +339,6 @@ final class _ManagedReference<Key: SharedReaderKey>: Reference, Observable {
 }
 
 extension _ManagedReference: MutableReference, Equatable where Key: SharedKey {
-  var wrappedValue: Key.Value {
-    base.wrappedValue
-  }
-
```

Comment:
Unrelated cleanup: this is already defined in the base class, so no need to duplicate that work in this extension.